auth/client/src/test/java/org/wildfly/security/auth/client/DefaultSSLContextNonexistentConfigFileTest.java

```diff
@@ -0,0 +1,34 @@
+package org.wildfly.security.auth.client;
```

Comment:
Same here.

Reply:
@fjuma Thank you for review. I added missing headers. I also switched values in AssertEqual calls because expected and actual values were switched.